cli_ftd/CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ cmake_minimum_required(VERSION 3.17)
 # set(TARGET_BOARD "CC1352P_2_LAUNCHXL")
 set(TARGET_BOARD "CC1352R1_LAUNCHXL")
 
+set(UART_BAUD_RATE 9600) # 115200 (console) or 9600 (bluetooth)
+set(BLUETOOTH 1) # 1 or 0
```

Comment:
This is the variable you check for BLUETOOTH configurations

Add it to your debugPrint

i.e if DEBUG_PRINT == 1 && BLUETOOTH == 0 (only then can you debug print messages)

Reply:
Got it.